tutorials/Tutorial11_Pipelines.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>Maybe add "One popular use case for such query classifiers is routing keyword queries to Elasticsearch and questions to DPR + Reader. With this approach you keep optimal speed and simplicity for keywords while going deep with transformers when it's most helpful."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>